BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -7,6 +7,7 @@ namespace BurnOutSharp.ProtectionType
 {
     public class AACS
     {
+	    // TODO: Detect version
```

Comment:
Please make sure that everything is aligned. This applies to everywhere that you converted tabs to spaces.